### ISSUE TITLE
Static predicate field names are inconsistent

### ISSUE THREAD

**Issue by haykam821** [asiekierka/MinecraftPOMF#1624]
There isn't a consistent name scheme for static predicate fields. We should decide on one of these formats:

- No prefix or suffix: [`RIDERS`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/item/BoatItem.mapping#L2)
- 'Is' or 'can' prefix only: [`IS_FOOD`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/entity/passive/PandaEntity.mapping#L6)
- 'Predicate' suffix: `AIR_PREDICATE`
- 'Is' or 'can' prefix and 'predicate' suffix: [`IS_AIR_PREDICATE`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/item/BoatItem.mapping#L2)
- 'Checker' suffix: [`DOOR_BREAK_DIFFICULTY_CHECKER`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/entity/mob/ZombieEntity.mapping#L2)
- 'Is' or 'can' prefix and 'checker' suffix: `IS_DOOR_BREAK_DIFFICULTY_CHECKER`
- 'Filter' suffix: [`BLOW_UP_FILTER`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/entity/passive/PufferfishEntity.mapping#L4)
- 'Is' or 'can' prefix and 'filter' suffix: `IS_BLOW_UP_FILTER`
- Just predicate: [`PREDICATE`](https://github.com/FabricMC/yarn/blob/9ba8dbc9dfb9bc7e3cb3d55d39c817dff3aafff1/mappings/net/minecraft/entity/decoration/AbstractDecorationEntity.mapping#L3)

**Comment by liach** [comment]
I prefer only `FILTER`/`CHECKER` suffix without `IS` prefix.

**Comment by YanisBft** [comment]
I would prefer `PREDICATE` suffix without `IS` prefix

**Comment by Earthcomputer** [comment]
Imo a predicate is a function, so `IS_FOOD` makes the most sense to me, and fits with the style I tend to go with in my personal code.

**Comment by TheBrokenRail** [comment]
I also prefer ``IS_FOOD`` or ``IS_FOOD_PREDICATE``

**Comment by Runemoro** [comment]
Yes, I also prefer the `IS_` suffix. More generally I think we should name `Predicate`s and `Function`s as if they were methods.

**Comment by liach** [comment]
> Yes, I also prefer the IS_ **suffix**.

![concern](https://cdn.discordapp.com/emojis/707016186281590855.png?v=1)

**Comment by Sollace** [comment]
> 'Is' or 'can' prefix only: IS_FOOD

I vote for `IS_` prefix only. It's just the most concise.

**Comment by liach** [comment]
looking back, now i support is prefix. or we can alternatively have a 3rd person present tense verb, like HAS_EFFECT, REQUIRES_SOME_PERMISSION, etc.
